modules/codebuild-dashboard/main.tf

```diff
@@ -0,0 +1,260 @@
+terraform {
+  required_providers {
+    lightstep = {
+      source  = "lightstep/lightstep"
+      version = "~> 1.60.2"
+    }
+  }
+  required_version = ">= v1.0.11"
+}
+
+resource "lightstep_metric_dashboard" "aws_codebuild_dashboard" {
+  project_name   = var.lightstep_project
+  dashboard_name = "AWS CODEBUILD"
+
+  chart {
+    name = "Duration"
+    rank = "0"
+    type = "timeseries"
+
+    query {
+      query_name = "g"
+      display    = "bar"
+      hidden     = false
+
+      metric              = "aws.codebuild.duration_count"
+      timeseries_operator = "delta"
+
+
+      group_by {
+        aggregation_method = "sum"
+        keys               = []
+      }
+
+    }
+
+    query {
+      query_name = "a"
+      display    = "bar"
+      hidden     = false
+
+      metric              = "aws.codebuild.builds_count"
+      timeseries_operator = "delta"
+
+
+      group_by {
+        aggregation_method = "sum"
+        keys               = []
```

Comment:
Should the `keys` slice contain some values?

Reply:
> Should the `keys` slice contain some values?



> Should the `keys` slice contain some values?

As for this one there is an additional implementation to handle `keys` values. Those are provided by Grafana dataset.

Reply:
What are these keys? Fascinating.